docs/diagrams/BetterModelClassDiagram.puml

```diff
@@ -16,6 +16,6 @@ Person -up-> "*" Tag
 
 Person *--> Name
 Person *--> Phone
```

Comment:
We'll need to update this to Postal Code

Reply:
ok, next time